resources/profiles/Creality/machine/Creality Ender-3 S1 0.4 nozzle.json

```diff
@@ -103,7 +103,7 @@
 	"default_filament_profile": [
 		"Creality Generic PLA"
 	],
-	"machine_start_gcode": "G90 ; use absolute coordinates\nM83 ; extruder relative mode\nM140 S{hot_plate_temp_initial_layer} ; set final bed temp\nM104 S150 ; set temporary nozzle temp to prevent oozing during homing\nG4 S10 ; allow partial nozzle warmup\nG28 ; home all axis\nG1 Z50 F240\nG1 X2 Y10 F3000\nM104 S{nozzle_temperature_initial_layer} ; set final nozzle temp\nM190 S{hot_plate_temp_initial_layer} ; wait for bed temp to stabilize\nM109 S{nozzle_temperature_initial_layer} ; wait for nozzle temp to stabilize\nG1 Z0.28 F240\nG92 E0\nG1 Y140 E10 F1500 ; prime the nozzle\nG1 X2.3 F5000\nG92 E0\nG1 Y10 E10 F1200 ; prime the nozzle\nG92 E0",
+	"machine_start_gcode": "G90 ; use absolute coordinates\nM83 ; extruder relative mode\nM140 S[hot_plate_temp_initial_layer] ; set final bed temp\nM104 S150 ; set temporary nozzle temp to prevent oozing during homing\nG4 S10 ; allow partial nozzle warmup\nG28 ; home all axis\nG1 Z50 F240\nG1 X2 Y10 F3000\nM104 S[nozzle_temperature_initial_layer] ; set final nozzle temp\nM190 S[hot_plate_temp_initial_layer] ; wait for bed temp to stabilize\nM109 S[nozzle_temperature_initial_layer] ; wait for nozzle temp to stabilize\nG1 Z0.28 F240\nG92 E0\nG1 Y140 E10 F1500 ; prime the nozzle\nG1 X2.3 F5000\nG92 E0\nG1 Y10 E10 F1200 ; prime the nozzle\nG92 E0",
```

Comment:
Better to use `bed_temperature_initial_layer_single` instead of `hot_plate_temp_initial_layer`

Reply:
@SoftFever 

Okay, thanks for pointing that out.

I had it tested with the _hot_plate_temp_initial_layer_ parameters and it always worked without problems. Also with all printing processes. I also just tested a print with this parameters _bed_temperature_initial_layer_single_. 

The parameter was changed from _hot_plate_temp_initial_layer_ to _bed_temperature_initial_layer_single_.

Reply:
@fr3ak2402 
hot_plate_temp_initial_layer only works if the hot plate bed type is selected.
This is fine for all non-Bambu printers at the moment since we don't allow users to change bed type.
But for future proof, we should use bed_temperature_initial_layer_single

Reply:
@SoftFever 
Okay, I will then adjust that for all printers